bin/python/ripple/build/module/Default.py

```diff
@@ -0,0 +1,215 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals)
+
+import os
+
+from beast.build.Module import Module
+
+from beast.build.Function import Env, compose, for_tags, not_tags
+
+Append = Env.Append
+Prepend = Env.Prepend
+
+
+MODULE = Module(
+    files=compose(
+        for_tags(
+            'debug',
+            Append(CPPDEFINES=['DEBUG', '_DEBUG']),
+        ),
+
```

Comment:
Second part.

Regarding Function and Env specifically... these do use "advanced" code to
implement, but I don't think we'll *ever* need to change it.

Client code that uses it is short
<https://github.com/rec/rippled/blob/scons/src/beast/python/beast/build/module/Beast.py#L15-L26>
and, I hope, understandable.

It is very rare that I actually import names unqualified - but at the end
of this I had it down to a tiny number of symbols that aren't likely to
grow much.


I'm actually going to go and cut these down even a little right now.  :-)